opa-audit-operator/tests/integration/test_charm.py

```diff
@@ -0,0 +1,119 @@
+import logging
```

Comment:
Github won't let me comment on empty files, so I'm commenting here instead.

Where are these files referenced? Can they be removed?

* opa-audit-operator/tests/data/empty
* opa-audit-operator/tests/unit/empty